assets/js/app/Layouts.js

```diff
@@ -354,28 +354,30 @@ LocusZoom.Layouts.add("data_layer", "association_pvalues", {
 });
 
 LocusZoom.Layouts.add("data_layer", "phewas_pvalues", {
+    namespace: {"phewas": "phewas"},
     id: "phewaspvalues",
-    type: "scatter",
+    type: "category_scatter",
     point_shape: "circle",
     point_size: 70,
     tooltip_positioning: "vertical",
-    id_field: "{{namespace}}id",
-    fields: ["{{namespace}}phewas"],
+    id_field: "{{namespace[phewas]}}id",
+    fields: ["{{namespace[phewas]}}id", "{{namespace[phewas]}}log_pvalue", "{{namespace[phewas]}}trait_group", "{{namespace[phewas]}}trait_label"],
     x_axis: {
-        field: "{{namespace}}x"
+        field: "{{namespace[phewas]}}x",  // Synthetic/derived field added by `category_scatter` layer
+        category_field: "{{namespace[phewas]}}trait_group"
     },
     y_axis: {
         axis: 1,
-        field: "{{namespace}}pval|neglog10",
+        field: "{{namespace[phewas]}}log_pvalue",
         floor: 0,
         upper_buffer: 0.1
```

Comment:
Minor, but the example feels a bit tight and upping this value (0.15 or 0.2) might give it a bit more breathing room.